pytket/pytket/qasm/qasm.py

```diff
@@ -1042,7 +1043,17 @@ def circuit_to_qasm_io(
         elif header == "hqslib1" and optype == OpType.ZZPhase:
             # special handling for zzphase
             opstr = "RZZ"
-            params = op.params
```

Comment:
What happens to the other parameters which are not the first element of the `op.params` list? Or is that only one element anyway?